src/agent/onefuzz/src/expand.rs

```diff
@@ -85,7 +86,7 @@ impl<'a> Expand<'a> {
 
     fn extract_file_name_no_ext(&self, _format_str: &str) -> Option<ExpandedValue<'a>> {
         match self.values.get(&PlaceHolder::Input.get_string()) {
-            Some(ExpandedValue::Scalar(fp)) => {
+            Some(ExpandedValue::Path(fp)) | Some(ExpandedValue::Scalar(fp)) => {
```

Comment:
Since you set `PlaceHolder::Input` to an `ExpandedValue::Path`, you shouldn't need the or case for Scalar here.